### ISSUE TITLE
Listing node-pool's has multiple entries for same UID

### ISSUE THREAD

**Issue by NerdyShawn** [civo/cli#384]
Added a net new Node-Pool to existing type=k3s cluster with a single node via UI today. The ui looks proper but the CLI for some reason was listing the same node twice.
```c
date && civo version && civo k8s node-pool ls test01
Fri Apr 19 08:43:55 AM EDT 2024
A newer version (v1.0.81) is available, please upgrade with "civo update"
Civo CLI v1.0.80

Node Pool 90ab1b3e-d68b-4f0b-b618-69e567535238:
+--------------------------------------+----------------+-------+--------+--------+
| Name                                 | Size           | Count | Labels | Taints |
+--------------------------------------+----------------+-------+--------+--------+
| 90ab1b3e-d68b-4f0b-b618-69e567535238 | g4s.kube.large |     1 | {}     | []     |
+--------------------------------------+----------------+-------+--------+--------+

Node Pool 676be50f-9fbe-4bc7-a578-f81c35f91466:
+--------------------------------------+-----------------+-------+--------+--------+
| Name                                 | Size            | Count | Labels | Taints |
+--------------------------------------+-----------------+-------+--------+--------+
| 676be50f-9fbe-4bc7-a578-f81c35f91466 | g4s.kube.xsmall |     1 | {}     | []     |
+--------------------------------------+-----------------+-------+--------+--------+
| 676be50f-9fbe-4bc7-a578-f81c35f91466 | g4s.kube.xsmall |     1 | {}     | []     |
+--------------------------------------+-----------------+-------+--------+--------+

```
I was behind a minor version on CLI, but bumping to the latest still displayed the same node UID twice.
```c
date && civo version && civo k8s node-pool ls test01
Fri Apr 19 08:44:52 AM EDT 2024
Civo CLI v1.0.81

Node Pool 90ab1b3e-d68b-4f0b-b618-69e567535238:
+--------------------------------------+----------------+-------+--------+--------+
| Name                                 | Size           | Count | Labels | Taints |
+--------------------------------------+----------------+-------+--------+--------+
| 90ab1b3e-d68b-4f0b-b618-69e567535238 | g4s.kube.large |     1 | {}     | []     |
+--------------------------------------+----------------+-------+--------+--------+

Node Pool 676be50f-9fbe-4bc7-a578-f81c35f91466:
+--------------------------------------+-----------------+-------+--------+--------+
| Name                                 | Size            | Count | Labels | Taints |
+--------------------------------------+-----------------+-------+--------+--------+
| 676be50f-9fbe-4bc7-a578-f81c35f91466 | g4s.kube.xsmall |     1 | {}     | []     |
+--------------------------------------+-----------------+-------+--------+--------+
| 676be50f-9fbe-4bc7-a578-f81c35f91466 | g4s.kube.xsmall |     1 | {}     | []     |
+--------------------------------------+-----------------+-------+--------+--------+
```
Here is the UI view
![image](https://github.com/civo/cli/assets/16088118/b209d99f-c870-4391-a718-b7b824b59f3e)

**Comment by NerdyShawn** [comment]
to add to this I've also tried in another region to bring up a gpu cluster with the nvidia-gpu-operator

which I suspect is contributing to me running into issues there with that creation as well as it timeouts after an hour trying to get a healthy cluster via the [Terraform](https://github.com/civo-learn/civo-gpu-operator-tf) example.
```c
civo k8s ls --region LON1
+--------------------------------------+-------------------+--------------+-------+-------+-------------------------------------+
| ID                                   | Name              | Cluster-Type | Nodes | Pools | Conditions                          |
+--------------------------------------+-------------------+--------------+-------+-------+-------------------------------------+
| a97f500c-8e27-425d-b79d-4f80466c1606 | gpu_operator_civo | talos        |     1 |     1 | All Workers Up: False               |
|                                      |                   |              |       |       | Cluster On Desired Version: Unknown |
|                                      |                   |              |       |       | Control Plane Accessible: Unknown   |
|                                      |                   |              |       |       |                                     |
+--------------------------------------+-------------------+--------------+-------+-------+-------------------------------------+
| a97f500c-8e27-425d-b79d-4f80466c1606 | gpu_operator_civo | talos        |     1 |     1 | All Workers Up: False               |
|                                      |                   |              |       |       | Cluster On Desired Version: Unknown |
|                                      |                   |              |       |       | Control Plane Accessible: Unknown   |
|                                      |                   |              |       |       |                                     |
+--------------------------------------+-------------------+--------------+-------+-------+-------------------------------------+
```

**Comment by alejandrojnm** [comment]
@uzaxirr @alessandroargentieri check this

**Comment by alessandroargentieri** [comment]
@NerdyShawn 👋 are you able to give it another try? We might have fixed it in the most recent CLI release (`v1.0.83`)

**Comment by NerdyShawn** [comment]
New version cleared it right up, thanks all! :partying_face: 

shows a single cluster now
```c
date && civo version && civo k8s ls
Thu May  9 08:51:40 AM EDT 2024
Civo CLI v1.0.83
+--------------------------------------+--------+--------------+-------+-------+----------------------------------+
| ID                                   | Name   | Cluster-Type | Nodes | Pools | Conditions                       |
+--------------------------------------+--------+--------------+-------+-------+----------------------------------+
| 002fb84e-063c-409f-a516-4c9fa016800d | test01 | k3s          |     1 |     1 | Control Plane Accessible: True   |
|                                      |        |              |       |       | All Workers Up: True             |
|                                      |        |              |       |       | Cluster On Desired Version: True |
|                                      |        |              |       |       |                                  |
+--------------------------------------+--------+--------------+-------+-------+----------------------------------+
```
single entry for node-pool as well
```c
civo k8s node-pool ls test01

Node Pool 90ab1b3e-d68b-4f0b-b618-69e567535238:
+--------------------------------------+----------------+-------+--------+--------+
| Name                                 | Size           | Count | Labels | Taints |
+--------------------------------------+----------------+-------+--------+--------+
| 90ab1b3e-d68b-4f0b-b618-69e567535238 | g4s.kube.large |     1 | null   | null   |
+--------------------------------------+----------------+-------+--------+--------+
```

**Comment by alessandroargentieri** [comment]
fixed in [#418](https://github.com/civo/cli/pull/418), CLI version v1.0.83
